lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/HttpRetryer.java

```diff
@@ -56,24 +53,33 @@ public CloseableHttpResponse sendHTTPRequestRetryIfAllowed(HttpUriRequest reques
             try {
                 httpResponse = httpClient.execute(request);
 
-                retry = shouldHttpClientRetry(httpResponse.getStatusLine().getStatusCode());
+                int statusCode = httpResponse.getStatusLine().getStatusCode();
+
+                retry = httpRetryStatusConfig.shouldHttpClientRetry(statusCode);
+
+                if (retry) {
+                    LOGGER.warn("shouldHttpClientRetry statusCode - {}", statusCode);
```

Comment:
Can we change this just so it explicitly says that we should retrye (I can see the log text causing confusion down the line)